docker/dockerfile.merlin

```diff
@@ -104,9 +104,9 @@ RUN pip install --no-cache-dir --upgrade pip; pip install --no-cache-dir "cmake<
 
 # cupy-cuda wheels come from a different URL on aarch64
 RUN if [ $(uname -m) == "aarch64" ] ; then \
-        pip install cupy-cuda115 -f https://pip.cupy.dev/aarch64/ ; \
+        pip install cupy-cuda11x -f https://pip.cupy.dev/aarch64/ ; \
     else \
-        pip install cupy-cuda117 ; \
+        pip install cupy-cuda11x ; \
```

Comment:
Maybe I'm missing something but shouldn't these be `cupy-cuda12x` not `cuda11x` if we want the minimum version to be 12.0?

Reply:
It's confusing, because CUDA and CuPy are both currently at version 12. We need CuPy version 12.0.0 or greater, which is available for CUDA 11.